mopidy_soundcloud/library.py

```diff
@@ -144,9 +151,24 @@ def lookup(self, uri):
             uri = uri.replace("sc:", "")
             return self.backend.remote.resolve_url(uri)
 
+        if "directory" in uri:
```

Comment:
I think a check along these lines would be better (more robust):

    if uri.startswith("sc:directory:"):

I'm not intimately familiar with the Soundcloud extension so this might need adjustment.

Reply:
I agree they can be made more robust, I'll push an update for it.

I'm not sure when 'sc:' is invoked when browsing (maybe deprecated?) so that's still left in there.